Bottom: b9d37e284d109812bcaa6482aa70795eaf452ebf
Top:    75faea28cbd6df0a462c998143ead171fb912412
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-10-04 19:24:14 +0100

Refresh of play-with-some-things

---

diff --git a/src/toporouter.c b/src/toporouter.c
index f026d28..be67f19 100644
--- a/src/toporouter.c
+++ b/src/toporouter.c
@@ -6503,7 +6503,7 @@ oproute_rubberband(toporouter_t *r, GList *path)
   path_set_oproute(path, oproute);
 
 //  if(!strcmp(oproute->netlist, "  unnamed_net1")) 
-  oproute_path_speccut(oproute);
+//  oproute_path_speccut(oproute);
 
   oproute->arcs =
     oproute_rubberband_segment (r, oproute, path,
